pygeos/test/test_geometry.py

```diff
@@ -135,10 +140,17 @@ def test_get_coordinate_dimension():
     actual = pygeos.get_coordinate_dimension([point, point_z, None]).tolist()
     assert actual == [2, 3, -1]
 
-
+# FIXME:
+@pytest.mark.xfail(sys.platform == "win32", reason="Not correctly detecting function signature")
```

Comment:
Is this now solved @brendan-ward ? Otherwise, an approach like in #92 could be taken: supply the function pointer along with the none_value in the ufuncs `data` parameter.